draft-ietf-quic-load-balancers.md

```diff
@@ -756,6 +751,45 @@ the servers.
 Note that future versions of QUIC might not have Retry packets, require
 different information in Retry, or use different packet type indicators.
 
+### Considerations for Non-Initial Packets
+
+Initial Packets are especially effective at consuming server resources
+because they cause the server to create connection state. Even when mitigating
+this load with Retry Packets, the act of validating an Initial Token and sending
+a Retry Packet is more expensive than the response to a non-Initial packet with
+an unknown Connection ID: simply dropping it.
+
+Nevertheless, a Retry Service in Active Mode might desire to shield servers
+from non-Initial packets that do not correspond to a previously admitted
+Initial Packet. This has a number of considerations.
+
+* If a Retry Service maintains no per-flow state whatsoever, it cannot
+distinguish between valid and invalid packets and MUST forward all non-Initial
```

Comment:
"valid and invalid packets" here refers to "valid and invalid non-Initial packets" right? Probably worth clarifying.